php/Terminus/User.php

```diff
@@ -73,12 +75,21 @@ public function sites($organization=null) {
     return $response['data'];
   }
 
-  public function getId() {
-    return $this->id;
+  public function instruments() {
+    $instruments       = new Instruments(array('user' => $this));
```

Comment:
This should be initialized in the constructor. Properties, especially properties that are collections, should always be initialized even if they are empty. This is to ensure that one doesn't try to invoke methods on `null`, expecting it to be a collection.